File: aoj/volume100/java/A10002.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int a = scan.nextInt();
        int b = scan.nextInt();

        int area = a*b;
        int perimeter = 2*a + 2*b;
        System.out.printf("%d %d\n", area, perimeter);
    }
}

File: aoj/volume100/scala/A10002.scala
object Main {
  def main(args: Array[String]) {
    val sc = new java.util.Scanner(System.in)
    val a = sc.nextInt
    val b = sc.nextInt
    val area      = a * b
    val perimeter = 2*a + 2*b
    println(s"$area $perimeter")
  }
}
